README.md

```diff
@@ -143,4 +145,281 @@ If you want to support a new platform, please consider submitting a PR to get th
          target/ipk/helium-gateway-<version>-<platform>.<ipk or deb>
          ```
 
+## Additional usage info
+
+The Helium Gateway application can be configured to suit your hardware/software environment in a variety of ways - either from the command line, using customisations to the `settings.toml` file or with environment variables. The following sections describe this functionality in more detail as well as more general information on how to use the application.
```

Comment:
```suggestion
The Helium Gateway application can be configured to suit your hardware/software environment in a variety of ways - either from the command line, using customizations to the `settings.toml` file or with environment variables. The following sections describe this functionality in more detail as well as more general information on how to use the application.
```

Reply:
As a Brit, it hurts to use "z" instead of "s"  😂😂